app/src/main/res/values/strings.xml

```diff
@@ -215,7 +215,7 @@
     <string name="done">Done!</string>
     <string name="failure">Failed!</string>
     <string name="hide_app_title">Hiding the Magisk app…</string>
-    <string name="open_link_failed_toast">No application found to open the link</string>
```

Comment:
Not consistent with "app" elsewhere. That a search was made is not relevant.

Reply:
It's also just fine the way it is.

Reply:
It is one more word to get an idea, and "No application found" could also be the application looking for something.

Reply:
This is well within standard error wordings though, e.g. "No file found" "Directory not found" "Missing file"